src/license_manager_agent_ops.py

```diff
@@ -152,6 +165,9 @@ def upgrade(self, version: str):
             # Start license-manager-agent
             self.license_manager_agent_systemctl("start")
 
+        # Clear cache dir after upgrade to avoid stale data
+        self.setup_cache_dir()
```

Comment:
This should also be triggered by upgrade.

Reply:
The line you commented is inside `def upgrade(self, version: str)`, so I didn't get what you mean.

Reply:
It's cleared in the install and in the upgrade, but it's missing in the configure_etc_default

Reply:
solved in [118a02d](https://github.com/omnivector-solutions/charm-license-manager-agent/pull/42/commits/118a02d1a83beee7676f11543867f02f0767955a)